packages/tpetra/core/src/Tpetra_Details_DistributorPlan.cpp

```diff
@@ -175,10 +178,13 @@ size_t DistributorPlan::createFromSends(const Teuchos::ArrayView<const int>& exp
   // numActive is the number of sends that are not Null
   size_t numActive = 0;
   int needSendBuff = 0; // Boolean
-
+  int badID = -1; // only used in debug mode
```

Comment:
If this is only used in debug mode, why not move this and the check inside the for loop down inside the `if (debug)`?  That would make it clear what's going on

Reply:
good idea